daemon/apollo/index.js

```diff
@@ -0,0 +1,48 @@
+const { ApolloServer, gql } = require('apollo-server')
+
+var search = require('../lib/search.js')
+var db = require('../lib/db.js')
+
+/*
+ * Implentation of a GraphQL server using the Apollow framework.
+ * See https://www.apollographql.com/server
+ *
+ */
+
+// Type definitions define the "shape" of the data and specify
+// which ways the data can be fetched from the GraphQL server.
+const typeDefs = gql`
+  type Listing {
+    id: String
+    name: String
+    description: String
+    price: Float
```

Comment:
At some point we will probably add `ListingType` and `ListingCategory` here

Reply:
Yes, agreed the schema needs a *lot* more thinking. This is just a prototype.

I have started putting some thoughts into it but I don't have yet something ready to share.
At a high level I think it may look like this:
 - A core listing object including things like: Type, category, description, currency+price, pics, location, fractional data
- Custom fields that partner can use to store whatever data specific to their listings/apps. Would be garbage in/out.